nectar/config.py

```diff
@@ -172,7 +172,10 @@ def _process_ssl_settings(self):
                 data_arg_os_handle, file_arg_value = tempfile.mkstemp(dir=self.working_dir,
                                                                       prefix=prefix)
 
-                os.write(data_arg_os_handle, data_arg_value)
+                try:
+                    os.write(data_arg_os_handle, data_arg_value)
+                except UnicodeEncodeError:
+                    os.write(data_arg_os_handle, data_arg_value.encode('ascii', 'replace'))
```

Comment:
I believe `ascii` encoding which is seen in the error is just because it's a default in Python but we likely want to write and read files as a UTF-8 ones, internally we will still expect unicode. So assuming we have `data_arg_value` in unicode, I suggest to encode it into UTF-8, like
` data_arg_value.encode('utf-8')`.

Reply:
If we encode it as `utf-8`, then we would also have to decode it as `utf-8` before reading it, wherever the file is being read.

Reply:
That is true, and I think that's what we want to do. utf8 comments in cert files are apparently normal, and we should be able to preserve that.